vos/vos/commands/vls.py

```diff
@@ -1,165 +1,164 @@
-        """Get the size of the terminal
+        for node in opt.node:
+            if not node.startswith('vos:'):
+                raise ArgumentError(opt.node, "Invalid node name: {}".format(node))
+            logging.debug("getting listing of: %s" % str(node))
+            info_list = client.get_info_list(node)
+
+            if sort_key:
+                try:
+                    sorted_list = sorted(info_list, key=lambda name: name[1][sort_key], reverse=not opt.reverse)
+                except:
+                    sorted_list = info_list
+                finally:
+                    info_list = sorted_list
+
+            for item in info_list:
+                name_string = item[0]
+                for col in columns:
+                    value = item[1].get(col, None)
+                    value = value is not None and value or ""
+                    if col in __LIST_FORMATS__:
+                        sys.stdout.write(__LIST_FORMATS__[col](value))
+                    if item[1]["permissions"][0] == 'l':
+                        name_string = "%s -> %s" % (name_string, item[1]['target'])
+                sys.stdout.write("%s\n" % name_string)
+    except Exception as ex:
+        exit_on_exception(ex)
 
-        @return: tuple(int, int) giving the row/column of the terminal screen.
-        """
 
-        def ioctl_gwinsz(fd):
-            """
-
-            @param fd: A file descriptor that points at the Screen Parameters
-            @return: the termios screeen structure unpacked into an array.
-            """
-            try:
-                import fcntl
-                import termios
-                import struct
-                this_cr = struct.unpack('hh', fcntl.ioctl(fd, termios.TIOCGWINSZ, '1234'))
-            except:
-                return None
-            return this_cr
-
-        cr = ioctl_gwinsz(0) or ioctl_gwinsz(1) or ioctl_gwinsz(2)
-
-        if not cr:
-            try:
-                td = os.open(os.ctermid(), os.O_RDONLY)
-                cr = ioctl_gwinsz(td)
-                td.close()
-            except:
-                pass
-        if not cr:
-            try:
-                cr = (os.environ['LINES'], os.environ['COLUMNS'])
-            except:
-                cr = (25, 80)
-        return int(cr[1]), int(cr[0])
-
-    def size_format(size):
-        """Format a size value for listing"""
-        try:
-            size = float(size)
-        except Exception as ex:
-            logger.debug(str(ex))
-            size = 0.0
-        if opt.human:
-            size_unit = ['B', 'K', 'M', 'G', 'T']
-            try:
-                length = float(size)
-                scale = int(math.log(length) / math.log(1024))
-                length = "%.0f%s" % (length / (1024.0 ** scale), size_unit[scale])
-            except:
-                length = str(int(size))
-        else:
-            length = str(int(size))
-        return "%12s " % length
+def get_terminal_size():
```

Comment:
hmm. I think get_terminal_size  was in use at one time.  Perhaps not any more.  The idea was to do column display of file names.  I think I couldn't get this working the way I wanted and so never used in production.